### DISCUSSION TITLE
Error: Cannot find module nuxt imported

### DISCUSSION THREAD

**Discussion by ipacs13** [nuxt/nuxt#20728] [Questions]
- Operating System: Windows_NT
- Node Version:     v16.16.0
- Nuxt Version:     3.5.0-28056056.7cb4c699
- Nitro Version:    2.4.0-28055094.65665a4
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      vue, ssr, serverMiddleware, meta, css, buildModules, plugins, vite, app
- Runtime Modules:  -
- Build Modules:    unplugin-icons/nuxt

**Action:**

1. Upgraded nuxt version to 3.5.0
2. Run `yarn build`

![image](https://user-images.githubusercontent.com/10388785/236879361-c45593f6-9fd1-4da2-9ff6-5c1765c54d84.png)

I've been stuck on this issue. I've been searching this error but no luck. Your help is much appreciated. :)

**Comment by warflash** [comment]
Nuxt 3.5 is still in PR draft phase. If you intend to use or test the edge release and notice an issue with it I recommend creating a quick [stackblitz](https://stackblitz.com/github/nuxt/starter/tree/v3/?file=README.md) showcasing the issue and opening an issue for it if it's reproducible 👍
